microsite/docs/GettingStarted.md

```diff
@@ -98,3 +101,30 @@ Note:
 - During install, Talisman may fail to add the pre-commit hook to this repository because one already exists for Husky. This is fine because it can still execute in the existing husky pre-commit hook, once installed.
 
 - During install, Talisman will also ask you for the directory of your git repositories. If you don't want to install Talisman in all your git repos, then cancel out at this step.
+
+
+## Run Cloud Carbon Footprint in an ephemeral workspace
+
+If you want to spin up an temporary development environment for Cloud Carbon Footprint, you can use Gitpod, an open source service for creating disposable environments to work on software projects.
+
+### Using the hosted service
+
+If you have an account on https://gitpod.io, you can open a workspace for the cloud carbon footprint project by following the link in your browser of the form `https://gitpod.io/#project-url-on-github`
+
+So for Cloud carbon footpring, you would visit:
+
+https://gitpod.io/#https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/
+
+This will run through the steps outlined in the documentation below install the necessary dependencies for the project, and open an instance of VS Code in your browser that you an work on immediately.
```

Comment:
```suggestion
This will run through the steps outlined in the documentation below and install the necessary dependencies for the project, then open an instance of VS Code in your browser that you can work on immediately.
```